Runtime/Scripts/MeshSyncServer.Properties.cs

```diff
@@ -173,19 +166,27 @@ void SendUpdatedProperties()
 #endif
                 }
 
-                if (needsClientSync)
-                {
+                if (needsPythonCallback) {
+#if VERBOSE_LOGS
+                    Debug.Log("Sending changes, needed python callback.");
+#endif
+                    needsPythonCallback = false;
+                    sendChanges         = true;
+
+                    m_server.RequestPythonCallback();
+                }
+                else if (needsClientSync) {
+#if VERBOSE_LOGS
+                    Debug.Log("Sending changes, needed client sync.");
```

Comment:
same here about using the [MeshSync] prefix

Reply:
Done